src/index.html

```diff
@@ -11,12 +11,114 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+
+      <nav class="nav">
```

Comment:
You should mix block with the header here
https://mate.academy/en/learn/css-advanced/bem?section=video&videoId=4936